packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -247,9 +247,16 @@ export class Builder extends EventEmitter {
     );
 
     const schemaTypesPath = getSchemaTypesPath(projectConfig, this.options);
-    const definition = printSchema(projectConfig.getSchema(), this.options);
-    await mkdirp(dirname(schemaTypesPath));
-    await writeFile(schemaTypesPath, definition);
+    const definitions = generateSchemaTypes(
+      projectConfig.getSchema(),
+      this.options,
+    );
+    await mkdirp(schemaTypesPath);
```

Comment:
Need to make sure the documentation is updated to indicate that this MUST BE a directory now, not a file.